components/06-examples/beheer-basis.hbs

```diff
@@ -0,0 +1,169 @@
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Delft</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Eindhoven</td>
+              <td>Alleen niet vertrouwelijke documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Tilburg</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Breda</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Maastricht</td>
+              <td>Alle documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Amsterdam</td>
+              <td>Alleen niet vertrouwelijke documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+            <tr>
+              <td>Gemeente Utrecht</td>
+              <td>Alleen niet vertrouwelijke documenten</td>
+              <td>
+                {{> '@button' type='button' modifier='link' label='Bewerk' icon='pencil' iconOnly=true }}
+                {{> '@button' type='button' modifier='link' label='Verwijder' icon='trash' iconOnly=true }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-12">
+        <nav aria-label="Pagina navigatie">
```

Comment:
Liever het pagination component aanroepen, nu heb je de markup gedupliceerd

Reply:
Top, dank voor het aanpassen. Wat ik hiervan nog herinner: ik kreeg het niet goed werkend via de config.yml, zodoende had ik uiteindelijk de markup gekopieerd. Ik zie nu hoe ik de parameters mee kan geven binnen de handlebars. Ga ik in het vervolg zo proberen.